### DISCUSSION TITLE
Noob question: where can I find flowbite.bundle.js?

### DISCUSSION THREAD

**Discussion by sp88011** [themesberg/flowbite#78] [Q&A]
Step 3 of the installation guide says
```

3. Include the main JavaScript file to make interactive elements work:
<script src="../path/to/@themesberg/flowbite/dist/flowbite.bundle.js"></script>
```

...but where can I get find this file (flowbite.bundle.js)? 

I've set up a next.js project and tailwind works great but I'm stuck using flowbite because I can't figure out where to get this file.

**Comment by zoltanszogyenyi** [comment]
Hey @sp88011,

`flowbite.bundle.js` has been renamed to `flowbite.js` as of the v1.3.2 release.

This should be available in your `node_modules/` folder after installing the package.
